osbs/build/build_request.py

```diff
@@ -524,16 +534,20 @@ def adjust_for_scratch(self):
         and should not be imported into Koji.
         """
         if self.scratch:
-            for when, which in [
-                    ("prebuild_plugins", "koji_parent"),
-                    ("postbuild_plugins", "tag_from_config"),
-                    ("postbuild_plugins", "compress"),  # only for Koji
-                    ("postbuild_plugins", "koji_upload"),
-                    ("postbuild_plugins", "fetch_worker_metadata"),
-                    ("exit_plugins", "koji_promote"),
-                    ("exit_plugins", "koji_import"),
-                    ("exit_plugins", "koji_tag_build"),
-            ]:
+            remove_plugins = [
+                ("prebuild_plugins", "koji_parent"),
+                ("postbuild_plugins", "compress"),  # only for Koji
+                ("postbuild_plugins", "koji_upload"),
+                ("postbuild_plugins", "fetch_worker_metadata"),
+                ("exit_plugins", "koji_promote"),
+                ("exit_plugins", "koji_import"),
+                ("exit_plugins", "koji_tag_build"),
+            ]
+
+            if not self.has_tag_suffixes_placeholder():
+                remove_plugins.append(("postbuild_plugins", "tag_from_config"))
```

Comment:
Hang on, if it's conditional on the new parameter does that mean we're currently settings tags from additional-tags for scratch builds?